src/engine/BMInterfaceGame.php

```diff
@@ -49,23 +48,43 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                if ($this->isTest) {
```

Comment:
Heads up: putting test flags in non-test code makes me cringe.  I'm not going to veto it in the spirit of keeping things moving, but ideally there'd be a better way to do this.